src/GravityFormsExtensions.php

```diff
@@ -278,11 +278,11 @@ function ( $conf ) use ( $confirmation ) {
 		$confirmation_fields = [
 			'confirmation'    => $default_confirmation['message'],
 			'share_platforms' => [
-				'facebook' => '1' === $default_confirmation['facebook'] ?? false,
-				'twitter'  => '1' === $default_confirmation['twitter'] ?? false,
-				'whatsapp' => '1' === $default_confirmation['whatsapp'] ?? false,
-				'native'   => '1' === $default_confirmation['native'] ?? false,
-				'email'    => '1' === $default_confirmation['email'] ?? false,
+				'facebook' => '1' === isset( $default_confirmation['facebook'] ) ? $default_confirmation['facebook'] : true,
```

Comment:
Correct if I'm wrong, but this condition will always return `true` or am I missing something?

Reply:
Yeah I made a mistake in the first implementation, the PR has been updated!